library/src/webapp/editor/ckextraplugins/audiorecorder/plugin.js

```diff
@@ -184,19 +184,3 @@ Audio.prototype.setObjectElement = function (e){
 		this.width = GetAttribute( e, 'width', this.width );
 		this.height = GetAttribute( e, 'height', this.height );
 };
-
-String.prototype.endsWith = function(str)
```

Comment:
Are those safe to remove?

Reply:
yes. endsWith is now on String OOTB, and it does the same thing.

Reply:
nice :+1:

Reply:
Also, it overwrites the endsWith for any other js on the page at the time :)

Reply:
Yes, that was my concern, but all good.